web/src/click_handlers.js

```diff
@@ -260,6 +263,7 @@ export function initialize() {
 
     $("body").on("click", ".reveal_hidden_message", (e) => {
         const message_id = rows.id($(e.currentTarget).closest(".message_row"));
+        assert(message_lists.current !== undefined);
```

Comment:
Same with this one -- asserts at the top of the block are more readable.

Reply:
Done.